Makefile

```diff
@@ -208,7 +208,7 @@ OPERATOR_SDK ?= $(LOCALBIN)/operator-sdk
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v5.3.0
-CONTROLLER_TOOLS_VERSION ?= v0.15.0
+CONTROLLER_TOOLS_VERSION ?= v0.16.1
 OPERATOR_SDK_VERSION ?= v1.30.0
```

Comment:
SDK can be bumped as well to latest - v1.36.1

Reply:
OK. I will update some of these after #425 is merged

Reply:
done